src/tests/mock_build/dictionary.ino

```diff
@@ -0,0 +1,17 @@
+#include <Arduino.h>
+#include <SPI.h>
```

Comment:
Remove the SPI.h and SD.h includes, since they're only used for disk based access.